tvm/cli.py

```diff
@@ -362,10 +362,15 @@ def projects() -> None:
 def init(name: str = None, version: str = None):
     """Configure a new tvm project in the current path."""
     if not version:
-        version = get_active_version()
+        version = version_manager.current_version(f"{TVM_PATH}")
+    if not version:
```

Comment:
Please remove this conditional, It's duplicated.

Reply:
Done